type_inference/ast_traverse.py

```diff
@@ -1,18 +1,67 @@
 import sys
```

Comment:
The file should start with Apache 2 comment. See Logica's files for an example. But Copyright Logica Authors, rather than Google.

Reply:
Same here